app/src/main/res/values/strings.xml

```diff
@@ -307,20 +307,20 @@
     <string name="tab_about">About</string>
     <string name="tab_contributors">Contributors</string>
     <string name="tab_licenses">Licenses</string>
-    <string name="app_description">A free lightweight YouTube frontend for Android.</string>
+    <string name="app_description">A libre lightweight YouTube frontend for Android.</string>
```

Comment:
We should generally think about replacing YouTube with streaming. SoundCloud is in beta yet, so this suffices for now.
Furthermore we should replace "fronted". NewPipe is more than this.
E.g. I use `NewPipe - the smart streaming solution` and `NewPipe the lightweight YouTube experience for Android` on https://newpipe.schabi.org

Reply:
"NewPipe, lightweight streaming enjoyment on Android" ?
You cant actually stream to others (yet).